profiles/models.py

```diff
@@ -1,3 +1,42 @@
+import os
+
+from django.contrib.auth.models import User
+from django.core.validators import RegexValidator
 from django.db import models
+from django.db.models.signals import post_save
+
+
+# Django provides a table called user that stores basic user information like username, password and email id.
+class Pcuser(models.Model):
```

Comment:
Yes @amittbhardwj these are 2 different methods and I found this one convenient so used this method.